packages/camera/camera_android/android/build.gradle

```diff
@@ -55,7 +55,7 @@ buildFeatures {
             // the Java heap size when running the Android unit tests. The following property here
             // sets the heap size to a size large enough to run the robolectric tests across
             // multiple SDK levels.
-            jvmArgs "-Xmx1g"
+            jvmArgs "-Xmx4G"
```

Comment:
This was flagged by the repo checks. These look to be `testOptions`. @stuartmorgan does this change need to be published to pub?

Reply:
Nope, just the tooling not knowing about the internals of the file structure.